cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -1787,6 +1788,15 @@ public String getVariable(String name) {
         return variables == null ? Collections.<String>emptySet() : variables.keySet();
     }
 
+    /** {@link CallbackKind}s supported by {@link AuthenticationCallbackHandler} */
+    private static final Set<CallbackKind> SUPPORTED_CALLBACK_KINDS = new HashSet<>();
```

Comment:
EnumSet.of could be used there. 
How can we know that we are listing all the supported callbacks and not missing some? A comment that would detail the mapping between kind and callback instances could help.